tensorflow_recommenders_addons/dynamic_embedding/python/keras/layers/dynamic_layer_normalization_test.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+
+import tensorflow as tf
+
+from tensorflow_recommenders_addons.dynamic_embedding.python.keras.layers.dynamic_layer_normalization import \
+    DynamicLayerNormalization
+
+
+class DynamicLayerNormalizationTest(tf.test.TestCase):
```

Comment:
We'd better to have one case testing the scenario in which the new layer works with tfra API like the demo shows.